todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/domain.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * -\-\-
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.example.android.architecture.blueprints.todoapp.statistics
+
+import com.example.android.architecture.blueprints.todoapp.data.Task
+
+sealed class StatisticsState {
```

Comment:
Hmm I think we should be consistent in how we define sealed classes. In all examples I've seen, cases of a sealed class are not defined as inner classes. That's what I followed with the other feature. What is your reasoning for doing it this way?

Reply:
In my PR, I defined the cases as inner classes so that I wouldn't break the import statements in the java files. For example java files that import `StatisticsState.loading` would break if you didn't define `Loading` as an inner class.